x509/extended_key_usage_gen.go

```diff
@@ -145,7 +160,14 @@ func generateIntegerConstants(oidToName map[string]OID) []byte {
 
 func generateNameConstants(oidToName map[string]OID) []byte {
 	buffer := bytes.Buffer{}
-	for _, oid := range oidToName {
+	// Create sorted slice of keys to ensure deterministic output
+	var keys []string
+	for k := range oidToName {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+	for _, k := range keys {
+		oid := oidToName[k]
```

Comment:
Seems like an name should come out of `oidToName` not an OID.

Reply:
This map has a confusing name, it looks like it maps raw OIDs (represented as strings) straight from the `zmap/constants` csv into the `OID` type. https://github.com/zmap/zcrypto/blob/master/x509/extended_key_usage_gen.go#L253

So it's more like `rawToOID` or something like that. I think that I'll rename the maps for clarity instead.

Reply:
(also `oid := oidToName[k]` was effectively what was happening before my PR when the range-based for loop was used directly on oidToName)